### PR TITLE
Protean transparency ability

### PR DESCRIPTION
**Pull request by Asher-42**
adds in ability to toggle transparency (slimey!)
## About The Pull Request
adding it an ability for proteans to make their body transparent so we can now have slime synthetics instead of making an entire species for it after talking to BM.
## Changelog
:cl:
add: Two abilities for protean that makes either their entire body transparent or just the chest.
/:cl:

### CODE REVIEW

**Comment by Asher-42** [comment]
Ready for review

File: modular_chomp/code/modules/mob/living/carbon/human/species/station/protean/protean_powers.dm [hunk 1]
```diff
@@ -705,3 +705,45 @@
...
 
 #undef PER_LIMB_STEEL_COST
+
+/mob/living/carbon/human/proc/chest_transparency_toggle()
+	set name = "transparency toggle (chest only)"
+	set category = "Abilities.Protean"
+	if(stat || world.time < last_special)
+		return
+	last_special = world.time + 50
+	var/temporgans = src.organs
```
Comment (Kashargul) [inline modular_chomp/code/modules/mob/living/carbon/human/species/station/protean/protean_powers.dm:715]: not really needed, the src.organs or just organs can go directly into the for loop

Comment (Asher-42) [reply modular_chomp/code/modules/mob/living/carbon/human/species/station/protean/protean_powers.dm]: fixed

**Comment by Asher-42** [comment]
removed temporgans and used src.organs

**Pull request merged**